data/scrapper/erplogin/erpSession.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import getpass
+
+ERP_HOME_URL = 'https://erp.iitkgp.ac.in/IIT_ERP3/' #to get the session cookie
+SECURITY_QUESTION_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/getSecurityQues.htm'
+LOGIN_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/auth.htm'
+GET_ACAD_TOKEN_URL = 'https://erp.iitkgp.ac.in/Acad/central_breadth_tt.jsp?action=second'
+
+
+class Session:
+    """
+    An erp session
+
+    Logs into erpr after prompting the user to enter details
+
+    
+    Class Attributes::
+    -self.sessionToken #the erp SessionToken or JSESSIONID (as named by erp)
+    -self.ssoToken    #the SSOToken obtained after loggin into erp
+    -self.academicToken #the academic token, useful for getting the data from erp
+
+    Basic Usage::
+
+    >>>from erplogin import erpSession
```

Comment:
Modify this to `from erplogin.session import ERPSession`

Reply:
Done.